wizeline/wize-q-front
# Bug - WIP-323 - Fix logout button and remove unused css

#### What does this PR do?
- Fix the logout and cross style (by third time… 🙄 )
- Remove some unused css

#### Where should the reviewer start?
`src/components/App.js`

#### How should this be manually tested?
1. Login, you'll see that the ugly cross ✖️  is now replaced by a fancy good-looking `logout`
2. Logout and ask a question, in the confirmation modal you'll see that styles match again.
3. :tada:

#### Any background context you want to provide?
The previous PR to fix this was made to `master` (my bad), that's why it was working in `prod` but not in `stage`.

#### What are the relevant tickets?
(Link to issues, related PR, JIRA issues, etc.)
Closes [JIRA: #323](https://wizeline.atlassian.net/browse/WIP-323)

#### Screenshots :camera:
<img width="1276" alt="screen shot 2017-07-12 at 12 54 01 pm" src="https://user-images.githubusercontent.com/15201480/28132236-d88ac814-6701-11e7-8aa2-1b58e2b08605.png">
<img width="635" alt="screen shot 2017-07-12 at 12 54 11 pm" src="https://user-images.githubusercontent.com/15201480/28132238-d88fc378-6701-11e7-918b-643edde2873f.png">
